example/src/main/java/io/netty/example/http2/helloworld/server/HelloWorldHttp2Handler.java

```diff
@@ -78,7 +79,11 @@ private void sendResponse(ChannelHandlerContext ctx, int streamId, ByteBuf paylo
         Http2Headers headers = new DefaultHttp2Headers().status(OK.codeAsText());
         encoder().writeHeaders(ctx, streamId, headers, 0, false, ctx.newPromise());
         encoder().writeData(ctx, streamId, payload, 0, true, ctx.newPromise());
-        ctx.flush();
+        try {
+            flush(ctx);
+        } catch (Http2Exception e) {
```

Comment:
@Scottmitch I would just catch `Http2Exception` as this is defined by the `flush(...)` method. Ok ?